### PR TITLE
Added mailers

### PR DESCRIPTION
**Pull request by jeremyFreeAgent**
Now you can configure several mailers.

### CODE REVIEW

File: DependencyInjection/Compiler/RegisterPluginsPass.php [hunk 1]
```diff
@@ -24,13 +24,18 @@ class RegisterPluginsPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
-        if (!$container->hasDefinition('swiftmailer.mailer')) {
+        if (!$container->findDefinition('swiftmailer.mailer') || !$container->getParameter('swiftmailer.mailers')) {
             return;
         }
 
-        $definition = $container->findDefinition('swiftmailer.transport');
-        foreach ($container->findTaggedServiceIds('swiftmailer.plugin') as $id => $args) {
```
Comment (lyrixx) [inline DependencyInjection/Compiler/RegisterPluginsPass.php]: That is a BC break. ping @jeremyFreeAgent

Comment (jeremyFreeAgent) [inline DependencyInjection/Compiler/RegisterPluginsPass.php]: What about registering `swiftmailer.plugin` services to each transport?

Comment (lyrixx) [inline DependencyInjection/Compiler/RegisterPluginsPass.php]: I don't know swift mailer enough ...

Comment (shouze) [inline DependencyInjection/Compiler/RegisterPluginsPass.php]: It wasn't looking like a BC break at the time we did that. @lyrixx have you experienced any breakage?

Comment (lyrixx) [inline DependencyInjection/Compiler/RegisterPluginsPass.php]: Wep, my plugin is not registered anymore. Anyway, I just updated the tag name ;)

**Pull request merged**